### PR TITLE
docs: update faq for turbomind so not found

### PR DESCRIPTION
**Pull request by zhyncs**
## Motivation

as titled

Hi @lzhangzz @irexyc @lvhan028 May you help review this pr? Thanks.

## Modification

as titled

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/en/faq.md [hunk 1]
```diff
@@ -20,7 +20,27 @@ It may have been caused by the following reasons.
...
+2. If you have installed it and still encounter this issue, it is probably because you are executing turbomind-related command in the root directory of lmdeploy source code. Switching to another directory will fix it.
+
+But if you are a developer, you often need to develop and compile locally. The efficiency of installing whl every time is too low. You can specify the path of lib after compilation through symbolic links.
+
+```shell
+# mkdir and build locally
+mkdir bld && cd bld && bash ../generate.sh && ninja $(nproc)
+
+# go to the lmdeploy subdirectory from bld and set symbolic links
+cd ../lmdeploy && ln -s ../bld/lib .
```
Comment (irexyc) [inline docs/en/faq.md:34]: Will `ninja $(nproc)` install the so file to `../lmdeploy` folder ? If not, there may not have `../lmdeploy` folder

Comment (zhyncs) [reply docs/en/faq.md:34]: > Will `ninja $(nproc)` install the so file to `../lmdeploy` folder ? If not, there may not have `../lmdeploy` folder

In the root directory of `lmdeploy`, there is a subdirectory named `lmdeploy` https://github.com/InternLM/lmdeploy/tree/main/lmdeploy

Comment (zhyncs) [reply docs/en/faq.md:34]: lmdeploy (root directory) -> bld (build directory) -> lmdeploy (sub directory)

In the subdirectory named `lmdeploy`, `lib` is a **symbolic link** to the `lib` folder under `bld`.

```
lmdeploy
├── bld
│   ├── bin
│   ├── build.ninja
│   ├── cmake
│   ├── CMakeCache.txt
│   ├── CMakeFiles
│   ├── cmake_install.cmake
│   ├── compile_commands.json
│   ├── CPackConfig.cmake
│   ├── CPackSourceConfig.cmake
│   ├── _deps
│   ├── examples
│   ├── lib
│   ├── src
│   ├── tests
│   ├── TurboMindConfig.cmake
│   └── TurboMindTargets.cmake
└── lmdeploy
    ├── api.py
    ├── archs.py
    ├── cli
    ├── __init__.py
    ├── legacy
    ├── lib -> ../bld/lib
    ├── lite
    ├── __main__.py
    ├── messages.py
    ├── model.py
    ├── __pycache__
    ├── pytorch
    ├── serve
    ├── tokenizer.py
    ├── turbomind
    ├── utils.py
    ├── version.py
    └── vl
```

**Review by irexyc — APPROVED** [review]

File: docs/en/faq.md [hunk 2]
```diff
@@ -20,7 +20,27 @@ It may have been caused by the following reasons.
 pip install lmdeploy[all]
 ```
 
-2. If you have installed it and still encounter this issue, it is probably because you are executing turbomind-related command in the root directory of lmdeploy source code. Switching to another directory will fix it
+If you want to install the nightly build of LMDeploy's whl package, you can download and install it from the latest release at https://github.com/zhyncs/lmdeploy-build according to your CUDA and Python versions. Currently the update frequency of whl is once a day.
+
+2. If you have installed it and still encounter this issue, it is probably because you are executing turbomind-related command in the root directory of lmdeploy source code. Switching to another directory will fix it.
+
+But if you are a developer, you often need to develop and compile locally. The efficiency of installing whl every time is too low. You can specify the path of lib after compilation through symbolic links.
```
Comment (lvhan028) [inline docs/en/faq.md:27]: Can `build.md` help with it? 
The `ninja install` can install the target lib to `lmdeploy/lib`. You don't need to create a symbolic link to it.

Comment (zhyncs) [reply docs/en/faq.md:27]: The `ninja install` command will `cp` the library to the location. When using a symbolic link, it only needs to be set once.

File: docs/zh_cn/faq.md [hunk 3]
```diff
@@ -16,12 +16,32 @@ pip install --upgrade mmengine
...
+如果您想安装 LMDeploy 预编译包的 nightly 版本，可以根据您的 CUDA 和 Python 版本从 https://github.com/zhyncs/lmdeploy-build 下载并安装最新发布的包。目前更新频率是每天一次。
+
 2. 如果已经安装了，还是出现这个问题，请检查下执行目录。不要在 lmdeploy 的源码根目录下执行 python -m lmdeploy.turbomind.\*下的package，换到其他目录下执行。
 
+但是如果您是开发人员，通常需要在本地进行开发和编译。每次安装 whl 的效率太低了。您可以通过符号链接在编译后指定 lib 的路径。
+
+```shell
+# 创建 bld 和进行本地编译
+mkdir bld && cd bld && bash ../generate.sh && ninja $(nproc)
+
```
Comment (irexyc) [inline docs/zh_cn/faq.md:32]: missing -j?

`ninja -j$(nproc)`

Comment (zhyncs) [reply docs/zh_cn/faq.md:32]: ok

**Review by lvhan028 — APPROVED** [review]

**Pull request merged**